mkdocs.yml

```diff
@@ -118,6 +120,7 @@ nav:
       - adr/ADR-5_no-more-bootstrap.md
       - adr/ADR-6_dynamic-config.md
       - adr/ADR-7_wsgi-server.md
+      - adr/ADR-8_go-transition.md
```

Comment:
Do we also want to link ADR9, so that it's visible and could have a higher chance of feedback?

Reply:
done.